src/command_modules/azure-cli-network/azure/cli/command_modules/network/commands.py

```diff
@@ -490,11 +490,9 @@ def _make_singular(value):
 
     with self.command_group('network nsg rule', network_nsg_rule_sdk) as g:
         g.command('delete', 'delete')
+        g.custom_command('list', 'list_nsg_rules', table_transformer=lambda x: [transform_nsg_rule_table_output(i) for i in x])
         g.show_command('show', 'get', table_transformer=transform_nsg_rule_table_output)
-        g.command('list', 'list', table_transformer=lambda x: [transform_nsg_rule_table_output(i) for i in x])
         g.custom_command('create', 'create_nsg_rule_2017_06_01', min_api='2017-06-01')
-        g.generic_update_command('update', setter_arg_name='security_rule_parameters', min_api='2017-06-01',
-                                 custom_func_name='update_nsg_rule_2017_06_01', doc_string_source='SecurityRule')
```

Comment:
if you meant to make this change, then you should remove the `update_nsg_rule_2017_06_01` from the custom.py as it appears not used any commands any more?

Reply:
Yeah, this is why the CI failed. I reverted the change.

Reply:
It resulted from the bad merge.